### ISSUE TITLE
When navigating words in an edit field in chrome, nvda does not report a line break

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9305]
### Steps to reproduce:
This problem can be reproduced when opening a call for nvda on nvda's own GitHub.
Open a call and go to the edit body comment field.

Walk with the arrows to the blank line below
Actual behavior:
enter a short text as an example
this is a test.
press enter ,.
The lines will look like this:

### Actual behavior:
this is a test.

### Expected behavior:

now go to the beginning of the line
this is a test
walk with ctrl + right arrow and watch when nvda reads:
test

### Actual behavior:
read nvda
test ###

see that the nvda read as if the word
test
was clinging to the symbols
###

### Expected behavior:
by pressing ctrl + right arrow the nvda announces
test
by pressing another ctrl + right arrow the nvda read
new line
by pressing another ctrl + right arrow the nvda read
###
The expected behavior is observed when having the same lines in the word.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
2018.4.1
#### Windows version:
10 17134
#### Name and version of other software in use when reproducing the issue:
chrome 72
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by hassanalawi** [comment]
yes that is still remaining when i want navigate in android studio editor word by word  some time cant unless i press control right arrow  several times quickly . my windows is 10 and nvda ver is 2019.1.1  this will be solved if i separate all punktuations from words but you know that is very difficult to do that when you writing a code  in an IDE .

**Comment by Adriani90** [comment]
Still reproducible in NVDA 2019.3 RC3 and Chrome 80.
Actual:
NVDA reports when pressing ctrl+right arrow following:
* Exptected
* beavior (###)
* behavior (###)
* Behavior (###) 
and so on.
It seems like NVDA is in a loop when punctuations or return carriages are involved while navigating word by word.
This is also reproducible in Firefox 72.0.2 but the loop is not that long like in Chrome 80.

Expected:
When pressing ctrl+right arrow, NVDA should report following:
* (###) 
* Expected
* Behavior
* Carriage return
and so on.
Thus, carriage return and one or more puctuation signs next to each other  should be treated like words when navigating word by word.

cc: @jcsteh, @feerrenrut

**Comment by Adriani90** [comment]
closing as duplicate of #3156 and referenced issues there. The discussion is there in a comprehensive detail.

**Comment by CyrilleB79** [comment]
Reopening as per https://github.com/nvaccess/nvda/issues/3156#issuecomment-2191027411.

Moreover, the current issue is about Chrome whereas #3156 is about Firefox.
